### PR TITLE
Added test suite for metadata sync check

### PR DESCRIPTION
**Pull request by udaysk23**
Added test suite for metadata sync check

Following is the log location of successful run
http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-TE88M9

Signed-off-by: uday kurundwade <ukurundw@redhat.com>

# Description

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

**Comment by udaysk23** [comment]
Following is the new log location of successful run with secondary site verification
http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-OGG434

File: suites/nautilus/rgw/tier-2_rgw_test-multisite-metadata-sync.yaml [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+          config:
+            ansi_config:
+              ceph_test: True
+              ceph_origin: distro
+              ceph_repository: rhcs
+              osd_scenario: lvm
+              osd_auto_discovery: False
+              journal_size: 1024
+              ceph_stable: True
+              ceph_stable_rh_storage: True
```
Comment (psathyan) [inline suites/nautilus/rgw/tier-2_rgw_test-multisite-metadata-sync.yaml:21]: Do we need these configuration parameters?

File: suites/nautilus/rgw/tier-2_rgw_test-multisite-metadata-sync.yaml [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
```
Comment (psathyan) [inline suites/nautilus/rgw/tier-2_rgw_test-multisite-metadata-sync.yaml:47]: Likewise, do we need these configuration parameters.

File: suites/pacific/rgw/tier-2_rgw_test-multisite-metadata-sync.yaml [hunk 3]
```diff
@@ -0,0 +1,264 @@
...
+      clusters:
+        ceph-pri:
+          config:
+            verify_cluster_health: true
+            steps:
+              - config:
+                  command: bootstrap
+                  service: cephadm
+                  args:
+                    registry-url: registry.redhat.io
```
Comment (psathyan) [inline suites/pacific/rgw/tier-2_rgw_test-multisite-metadata-sync.yaml:22]: Let's avoid this key/value pair as in IBM, we use a private registry

File: suites/pacific/rgw/tier-2_rgw_test-multisite-metadata-sync.yaml [hunk 4]
```diff
@@ -0,0 +1,264 @@
...
+                        - node5
+        ceph-sec:
+          config:
+            verify_cluster_health: true
+            steps:
+              - config:
+                  command: bootstrap
+                  service: cephadm
+                  args:
+                    registry-url: registry.redhat.io
```
Comment (psathyan) [inline suites/pacific/rgw/tier-2_rgw_test-multisite-metadata-sync.yaml:66]: Likewise, let's avoid this key/value pair.

**Review by psathyan — COMMENTED** [review]
Looks good...

Some minor questions. It would be great if you could add the config to be used for the suite and the objective of the suite as comments.

**Review by psathyan — APPROVED** [review]

**Pull request merged**